### ISSUE TITLE
Information on ratelimit-string

### ISSUE THREAD

**Issue by mallorbc** [laurentS/slowapi#115]
Throughout the code and docs, it says to refer to ratelimit-string for more information on what are valid inputs.  However, I am unable to find information on this and have just guessed so far for my usage.  I would appreciate information on all the options

**Comment by laurentS** [comment]
Hi @mallorbc sorry this was confusing. Is this link what you're after? https://limits.readthedocs.io/en/stable/quickstart.html#rate-limit-string-notation
Let me know if this helps. We'll have to update the docs to include this, and maybe some examples.

**Comment by laurentS** [comment]
@mallorbc Did the above link solve your problem?

**Comment by mallorbc** [comment]
Yes thank you!  Perhaps adding a link to the README may be a good idea for others?
